domain/host/host.go

```diff
@@ -156,8 +156,6 @@ func Build(ip string, rpcport string, poolid string, ipAddrs ...string) (*Host,
 	host.ServiceD.Date = servicedversion.Date
 	host.ServiceD.Buildtag = servicedversion.Buildtag
 
-	*host = *host
-
```

Comment:
What in the world? I'd like to double check with someone to make sure nothing fancy is going on here before we take it out.